lib/sufia/configuration.rb

```diff
@@ -84,10 +84,14 @@ def audit_user_key
       @audit_user_key ||= 'audituser@example.com'
     end
 
-    # TODO: this is called working_path in curation_concerns
```

Comment:
I think I see. Would it be more acceptable to leave the upload_path method as is (not rename) and just add the cache_path method?